cli/cdsctl/group_token.go

```diff
@@ -14,23 +16,80 @@ var (
 
 	groupToken = cli.NewCommand(groupTokenCmd, nil,
 		[]*cobra.Command{
+			cli.NewListCommand(groupTokenListCmd, groupTokenListRun, nil),
 			cli.NewGetCommand(groupTokenCreateCmd, groupTokenCreateRun, nil),
+			cli.NewDeleteCommand(groupTokenDeleteCmd, groupTokenDeleteRun, nil),
 		})
 )
 
 var groupTokenCreateCmd = cli.Command{
 	Name:  "generate",
 	Short: "Generate a new token",
+	Long: `
+		Useful to generate a new token when you use the cli or the api in scripts or for your worker, hatchery, uservices.
+		The expiration must be [daily|persistent|session]
+		Daily expirate after one day
+		Persistent doesn't expirate until you revoke them
+		Pay attention you must be an administrator of the group to launch this command
+	`,
 	Args: []cli.Arg{
 		{Name: "groupname"},
 		{Name: "expiration"},
 	},
+	OptionalArgs: []cli.Arg{
+		{Name: "description"},
+	},
 }
 
 func groupTokenCreateRun(v cli.Values) (interface{}, error) {
-	token, err := client.GroupGenerateToken(v["groupname"], v["expiration"])
+	token, err := client.GroupGenerateToken(v["groupname"], v["expiration"], v.GetString("description"))
 	if err != nil {
 		return nil, err
 	}
 	return *token, nil
 }
+
+var groupTokenDeleteCmd = cli.Command{
+	Name:  "delete",
+	Short: "Delete a token linked to a group",
+	Long: `
+		Useful to delete a token from a group and so revoke it to unauthorize future connection
```

Comment:
don't put space at the beginning of the line. With space -> it will be displayed with a black section